frontend/src/metabase/lib/analytics.js

```diff
@@ -41,6 +41,22 @@ export const trackStructEvent = (category, action, label, value) => {
 };
 
 export const trackSchemaEvent = (schema, version, data) => {
+  // eslint-disable-next-line no-undef
+  if (process?.env?.NODE_ENV === "development") {
```

Comment:
Can we not do this by default, but require e.g. an env variable to be set up for it?
The console is already a pain to work with due to React prop-types errors.

Reply:
Good point, wrapped it in a env var